unity-renderer/Assets/Rendering/Culling/CullingController.cs

```diff
@@ -18,30 +19,33 @@ public class CullingController : ICullingController
     {
         private const string ANIMATION_CULLING_STATUS_FEATURE_FLAG = "animation_culling_status";
         private const bool DRAW_GIZMOS = false;
-        internal List<CullingControllerProfile> profiles = null;
+        
+        // Cache to avoid allocations when getting names
+        private readonly HashSet<Shader> avatarShaders = new HashSet<Shader>();
+        private readonly HashSet<Shader> nonAvatarShaders = new HashSet<Shader>();
 
-        private CullingControllerSettings settings;
+        public UniversalRenderPipelineAsset urpAsset;
+        
+        internal List<CullingControllerProfile> profiles = null;
 
         private HashSet<Renderer> hiddenRenderers = new HashSet<Renderer>();
         private HashSet<Renderer> shadowlessRenderers = new HashSet<Renderer>();
-
-        public UniversalRenderPipelineAsset urpAsset;
-
-        public ICullingObjectsTracker objectsTracker { get; private set; }
+        
+        private CullingControllerSettings settings;
         private Coroutine updateCoroutine;
         private float timeBudgetCount = 0;
         private bool resetObjectsNextFrame = false;
         private bool playerPositionDirty;
         private bool objectPositionsDirty;
         private bool running = false;
 
-        // Cache to avoid allocations when getting names
-        private readonly HashSet<Shader> avatarShaders = new HashSet<Shader>();
-        private readonly HashSet<Shader> nonAvatarShaders = new HashSet<Shader>();
+        public event ICullingController.DataReport OnDataReport;
```

Comment:
these new events are not being used

Reply:
@lorux0 I decided not to modify them, if you check the changes, you will see that all of them existed already, I just reordered them in order to comply with our code guidelines, so, in this case, there are no changes at all on this code, just reorder.